gnomad/sample_qc/ancestry.py

```diff
@@ -149,6 +150,7 @@ def assign_population_pcs(
 
     :param pop_pca_scores: Input Hail Table or Pandas Dataframe
     :param pc_cols: Columns storing the PCs to use
+    :param pc_names: List of integers to use for naming the selected PCs (i.e. an input of [1, 3] will result in the first two PCs of pc_cols being named PC1 and PC3)
```

Comment:
I think this certainly works, but I'm wondering if there is a better way to do this.  `pop_pca_scores` has all the PCA scores, and is filtered for use with `pc_cols`. I think there is likely a way that we can use this info to give the needed PC names rather than pass in names as another parameter?

Reply:
Do you mean change the pc_cols parameter? 
assign_population_pcs is currently called with 
```
pop_ht, pops_rf_model = assign_population_pcs(
            pop_pca_scores_ht,
            pc_cols=[pop_pca_scores_ht.scores[i - 1] for i in pcs],
            ....
```
so some information is being lost by just subsetting to these PCs for the Hail Table option

Reply:
Yeah, if we are already subsetting to these specific PCs, it seems like we can do it within the function. What I worry about is that we will add incorrect names due to this extra parameter, when we can modify this function so that we extract that info from the list, but I'm completely open to other thoughts

Reply:
I put in one way to modify this, let me know if it seems like a good option